### PR TITLE
Remove suggestion to change the `.class` parameters

### PR DESCRIPTION
**Pull request by mpdude**
To my knowledge, this practice has been discouraged and the corresponding configuration hooks have been dropped in Symfony 3.0.

### CODE REVIEW

File: bundles/override.rst [hunk 1]
```diff
@@ -37,39 +37,10 @@ If the controller is a service, see the next section on how to override it.
...
-        // app/config/config.php
-        $container->setParameter('translator.class', 'Acme\HelloBundle\Translation\Translator');
-
-Secondly, if the class is not available as a parameter, you want to make sure the
-class is always overridden when your bundle is used or if you need to modify
-something beyond just the class name, you should use a compiler pass::
+If you want to modify service definitions of another bundle, you can use a compiler
+pass to change the class of the service or to modify method calls. In the following
+example, the implementing class for the ``original-service-id`` is changed to 
+``Acme\DemoBundle\YourService`.
```
Comment (xabbuh) [inline bundles/override.rst]: this is missing the second backtick after the class name

File: bundles/override.rst [hunk 2]
```diff
@@ -37,39 +37,10 @@ If the controller is a service, see the next section on how to override it.
...
-        // app/config/config.php
-        $container->setParameter('translator.class', 'Acme\HelloBundle\Translation\Translator');
-
-Secondly, if the class is not available as a parameter, you want to make sure the
-class is always overridden when your bundle is used or if you need to modify
-something beyond just the class name, you should use a compiler pass::
+If you want to modify service definitions of another bundle, you can use a compiler
+pass to change the class of the service or to modify method calls. In the following
+example, the implementing class for the ``original-service-id`` is changed to 
+``Acme\DemoBundle\YourService``.
```
Comment (xabbuh) [inline bundles/override.rst]: Oh, we need to end this paragraph with two colons so that the following paragraph is actually interpreted as a code block.

**Comment by xabbuh** [comment]
For the records, this will fix #7093.

**Comment by wouterj** [comment]
👍 (except that we don't support 3.0 anymore, I suggest to merge it straight into 2.7)
status: reviewed

**Review by wouterj — APPROVED** [review]

**Comment by mpdude** [comment]
@wouterj wait, I think for 2.7 it's still ok. Pick a 3.x branch of your choice.

Should we add a note on 2.x that this is discouraged?

**Comment by mpdude** [comment]
Also, would it help to add a pointer to decorated services, or do they solve another problem?

**Comment by mpdude** [comment]
Possible duplicate of #6927 and #6912

**Comment by xabbuh** [comment]
Thank you @mpdude.

**Comment by xabbuh** [comment]
@mpdude Mentioning service decoration here sounds useful (a short paragraph linking to the existing chapter should be enough).

**Pull request closed without merging**